haskell/private/haskell_impl.bzl

```diff
@@ -358,11 +359,33 @@ def haskell_library_impl(ctx):
     ]
 
 def haskell_import_impl(ctx):
+    hs = haskell_context(ctx)
+
     if ctx.attr.package:
         package = ctx.attr.package
     else:
         package = ctx.label.name
-    return [HaskellPrebuiltPackageInfo(package = package)]
+
+    id_file = hs.actions.declare_file(target_unique_name(hs, "id"))
+    hs.actions.run_shell(
+        inputs = [hs.tools.ghc_pkg],
+        outputs = [id_file],
+        command = """
+        $1 --simple-output -v1 field $2 id > $3
```

Comment:
I’d add quotes around the variables here as well.

Reply:
Done